components/kyma-environment-broker/Makefile

```diff
@@ -36,7 +36,7 @@ go-mod-check: go-mod-check-local
 go-mod-check-local:
 	@echo make go-mod-check
 	go mod tidy
-	@if [ -n "$$(git status -s go.*)" ]; then \
+	@if [ -z "$$(git status -s go.*)" ]; then \
```

Comment:
this would introduce the bug with pipelines back - https://github.com/kyma-project/control-plane/pull/774